src/main/java/org/logstash/plugins/inputs/http/util/SslSimpleBuilder.java

```diff
@@ -116,4 +131,24 @@ private boolean requireClientAuth() {
 
         return false;
     }
+
+    public static String[] getDefaultCiphers(){
+        if (isUnlimitedJCEAvailable()){
+            return DEFAULT_CIPHERS;
+        } else {
+            logger.warn("JCE Unlimited Strength Jurisdiction Policy not installed - max key length is 128 bits");
```

Comment:
maybe we only need this warning? the one in line 73 seems redundant, or is it a different setting/code path?

Reply:
The intent of the warning on line 73 was to generate log a local warning/error along with the exception being thrown that stops the plugin from starting if no ciphers are available, whereas this is a general warning that lower strength keys will be used